sdk/eventhub/azure-eventhub/azure/eventhub/_eventprocessor/event_processor.py

```diff
@@ -197,6 +188,15 @@ def _create_tasks_for_claimed_ownership(self, claimed_partitions, checkpoints=No
                         try:
                             self._partition_initialize_handler(self._partition_contexts[partition_id])
                         except Exception as err:  # pylint:disable=broad-except
```

Comment:
This is old code. But could you insert this before here?
```python
except asyncio.CancelledError:
    raise
```

Reply:
This is sync code - so not needed here - but I have added a couple to other ambiguous places in the async version.

Reply:
ah yes, wrong comment place. I meant async